### DISCUSSION TITLE
PPRCODE中代码运行成功但结果报告中并无任何信息

### DISCUSSION THREAD

**Discussion by cuilinna123** [YaoYinYing/PPRCODE_Guideline#4] [Q&A] [answered]
PPRCODE中代码运行成功但结果报告xlsx表中并无任何信息，且网页中无任何图像展示，请问是什么原因呢？

**Comment by YaoYinYing** [comment, marked as answer]
> PPRCODE中代码运行成功但结果报告xlsx表中并无任何信息，且网页中无任何图像展示，请问是什么原因呢？
> 

您好，邮件已答复。
